src/App.js

```diff
@@ -0,0 +1,19 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import fetchGreeting from './redux/message/messageThunk';
+
+function App() {
+  const { greeting } = useSelector((state) => state.messages.message);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchGreeting());
+  }, [dispatch]);
+  return (
+    <h1>
+      {greeting}
+    </h1>
+  );
+}
+
```

Comment:
There are some problems with your implementations. Kindly check below. ⬇️ 

- [ ] It will be great to ensure that the app component has routes created by using react-router. This route can be `/greetings`. Then when this route is hit, it renders the greetings component.
- [ ] It will be great to create a separate greeting component that is responsible for fetching and displaying your messages.

Reply:
- [ ] @atok624 as previously requested kindly move the functionality to display a greeting to a new component `Greeting component`  and add a route for it using react-router in the app component, these are project requirements as shown below
![image](https://github.com/atok624/hello_react_front_end/assets/61408860/533052d9-0215-45a2-bfc7-43159f00da06)